src/Console/ModelsCommand.php

```diff
@@ -151,6 +154,7 @@ protected function getOptions()
           array('filename', 'F', InputOption::VALUE_OPTIONAL, 'The path to the helper file', $this->filename),
           array('dir', 'D', InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'The model dir', array()),
           array('write', 'W', InputOption::VALUE_NONE, 'Write to Model file'),
+          array('write-mixin', 'M', InputOption::VALUE_NONE, 'Write mixin to Model file'),
```

Comment:
(untested) but I think it makes sens to be clear how this works
```suggestion
          array('write-mixin', 'M', InputOption::VALUE_NONE, 'Write models to {$this->filename} and adds @mixin to each model, avoiding IDE duplicate declaration warnings'),
```